source/include/azure_iot_hub_client.h

```diff
@@ -30,7 +33,18 @@
 #define azureiothubSUBSCRIBE_FEATURE_COUNT    ( 3 )
 
 /* Forward declaration for Azure IoT Hub Client */
-typedef struct AzureIoTHubClient AzureIoTHubClient_t;
+typedef struct AzureIoTHubClient   AzureIoTHubClient_t;
+
+/**
+ * @brief Type for Azure IoT Plug and Play component. The list of component names can be set
+ * as an option for the Azure IoT Hub Client
+ */
+typedef az_span                    AzureIoTHubClientComponent_t;
```

Comment:
>                  [](http://example.com/codeflow?start=16&length=17)

nit: please format this properly ... either add one space or make all of the typedef align

Also how about having all MARCO first and then all typedef?

Reply:
Can do the macro one but otherwise it's the uncrustify at work